stl/inc/ranges

```diff
@@ -636,12 +636,138 @@ namespace ranges {
         /* [[no_unique_address]] */ _Ty _Value{};
     };
 
-    // CLASS TEMPLATE ranges::empty_view
+    // CLASS TEMPLATE ranges::_Non_propagating_cache
     // clang-format off
-    template <class _Ty>
-        requires is_object_v<_Ty>
-    class empty_view : public view_interface<empty_view<_Ty>> {
+    template <_Destructible_object _Ty>
+    class _Non_propagating_cache { // a simplified optional that resets on copy / move
+        // clang-format on

```

Comment:
This was suppressing clang-format due to the `requires`. Now that it's in the template parameter, I believe we can restore clang-format here.